src/hotspot/os/linux/gc/z/zMountPoint_linux.cpp

```diff
@@ -61,11 +62,12 @@ char* ZMountPoint::get_mountpoint(const char* line, const char* filesystem) cons
       strcmp(line_filesystem, filesystem) != 0 ||
       access(line_mountpoint, R_OK|W_OK|X_OK) != 0) {
     // Not a matching or accessible filesystem
-    os::free(line_mountpoint);
+    // sscanf, using %m, will return malloced memory. Need raw ::free, not os::free.
```

Comment:
The comment could be lifted to before sscanf so that it covers both uses of ::free.